SU2_CFD/include/fluid/CCoolPropViscosity.hpp

```diff
@@ -0,0 +1,141 @@
+/*!
+ * \file CCoolPropViscosity.hpp
+ * \brief Defines CoolPropviscosity model.
+ * \author P.Yan, G. Gori, A. Guardone,
+ * \version 7.4.0 "Blackbird"
+ *
+ * SU2 Project Website: https://su2code.github.io
+ *
+ * The SU2 Project is maintained by the SU2 Foundation
+ * (http://su2foundation.org)
+ *
+ * Copyright 2012-2022, SU2 Contributors (cf. AUTHORS.md)
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "CCoolProp.hpp"
+#include "CViscosityModel.hpp"
+
+#ifdef USE_COOLPROP
+#include "AbstractState.h"
+#include "CoolProp.h"
+#endif
+/*!
+ * \class CCoolPropViscosity
+ * \brief Defines CoolProp viscosity model.
+ * \author P.Yan
+ */
+class CCoolPropViscosity final : public CViscosityModel {
+ private:
+#ifdef USE_COOLPROP
+  std::unique_ptr<CoolProp::AbstractState> fluid_entity; /* \brief fluid entity */
+#endif
+  /* \brief list of fluids whose viscosity model is available in CoolProp */
+  vector<string> fluidNameList = {"Air",
```

Comment:
Please try re-running clang-format after this change, hopefully, it will make the list more concise like in the other class.
```suggestion
  std::unique_ptr<CoolProp::AbstractState> fluid_entity; /*!< \brief fluid entity */
#endif
  /*!< \brief list of fluids whose viscosity model is available in CoolProp */
  const vector<string> fluidNameList = {"Air",
```

Reply:
sorry, the list keeps unchanged after I added these changes.
I checked the ClangFormat options for CCoolPropConductivity and  CCoolPropViscosity, the options are same.
The only difference is the two lists with different fluid names.